ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -96,7 +96,7 @@ port_profiles:
         # Adapter speed - if not specified will be auto.
       - speed: < interface_speed | forced interface_speed | auto interface_speed >
 
-        # Local endpoint port(s)
+        # Local endpoint port(s) | required
         endpoint_ports: [ < interface_name > ]
 
         # List of port(s) connected to switches
```

Comment:
Could you please add the "| required" tag here and also in next variable, "switches"?